.github/workflows/rename-module.yml

```diff
@@ -60,22 +59,18 @@ jobs:
           go build ./...;
           go test ./accounts/abi/bind ./rlp/rlpgen
 
-      - name: Commit to new branch
-        uses: devops-infra/action-commit-push@8bc2ff9f9de7aa2a7581fc7e5b6401c04cab54c7
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          target_branch: ${{ env.output_branch }}
-          force: true
-          commit_prefix: "[AUTO] rename Go module + update internal import paths"
+      - name: Create new branch
+        env:
+          BRANCH: ${{ steps.vars.outputs.DEST_BRANCH }}
+        run: |
+          git checkout -b "${BRANCH}";
+          git push origin "${BRANCH}";
 
-      - name: Open PR to "renamed-go-module" iff workflow dispatched on "main"
-        # If we are changing the way in which we manage module renaming then it
-        # MUST go through PR review to main; only then can it open PRs.
-        if: github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main'
-        uses: devops-infra/action-pull-request@v0.5.5
+      - name: Commit to new branch
+        uses: planetscale/ghcommit-action@d4176bfacef926cc2db351eab20398dfc2f593b5 # v0.2.0
```

Comment:
curious why not refer to v0.2.0?

Reply:
It's possible to change a version tag to a different commit so this just reduces the attack surface.